app/user/views.py

```diff
@@ -18,16 +16,18 @@ def login():
     return render_template("user/login.html", page_title=title, form=login_form)
 
 
-@blueprint.route("/signup")
+@blueprint.route("/signup", methods=["POST", "GET"], endpoint='registration')
 def registration():
     if current_user.is_authenticated:
-        return redirect(url_for('index'))
+        return redirect(url_for("index"))
     title = "Регистрация"
     reg_form = RegistrationForm()
+    if request.method == "POST" and reg_form.validate():
```

Comment:
Кажется достаточно `if form.validate_on_submit():`

Reply:
Да в 
https://flask-wtf.readthedocs.io/en/0.15.x/quickstart/#validating-forms 
говорится что if form.validate_on_submit() под капотом имеет == POST
но тогда пропадает Дзен python 'явное лучше неявного' или это про другое? :))
```suggestion
    if request.method == "POST" and reg_form.validate():
```